app/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,204 @@
 <resources>
-    <string name="app_name">Grinnell Directory</string>
-    <string name="title_activity_default_login">Sign in</string>
+<string name="app_name">Grinnell Directory</string>
+<string name="title_activity_default_login">Sign in</string>
+
```

Comment:
@LarryAsante is it better to put these in here, in a `Constants.java` file, or put it on a server and pull it to make it easier to change?